rubin_sim/selfcal/solver.py

```diff
@@ -14,19 +14,20 @@ class LsqrSolver:
 
     Parameters
     ----------
-    observaitons : np.array
-        A numpy array of the observations. Should have columns id, patch_id, observed_mag, mag_uncert
-    patch_out : str ("solved_patches.npz")
+    observations : `np.array`
+        A numpy array of the observations.
+        Should have columns id, patch_id, observed_mag, mag_uncert
+    patch_out : `str` ("solved_patches.npz")
```

Comment:
Put the defaults in descriptions, or I think you'll confuse sphinx

Reply:
Fixed here. I'm hoping that it doesn't confuse sphinx too much, as we have these hanging around in many places still. I'm removing as I can, but it's a lot.